modules/oadp-release-notes-1-2-5.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes-1-2.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="migration-oadp-release-notes-1-2-5_{context}"]
+= OADP 1.2.5 release notes
+
+{oadp-first} 1.2.5 is a Container Grade Only (CGO) release, released to refresh the health grades of the containers, with no changes to any code in the product itself compared to that of {oadp-short} 1.2.4.
+
+
+[id="resolved-issues-1-2-5_{context}"]
+== Resolved issues
+
+// There are no resolved issues in {oadp-short} 1.2.5.
+
+.CVE-2023-2431: `oadp-velero-plugin-for-microsoft-azure-container`: Bypass of seccomp profile enforcement
+
+A flaw was found in Kubernetes, which impacts earlier versions of {oadp-short}. This flaw arises when Kubernetes allows a local authenticated attacker to bypass security restrictions, caused by a flaw when using the localhost type for a `seccomp` profile but specifying an empty profile field. An attacker can bypass the `seccomp` profile enforcement by sending a specially crafted request. This flaw has been resolved in {oadp-short} 1.2.5.
+
+For more details, see link:https://access.redhat.com/security/cve/CVE-2023-2431[(CVE-2023-2431)]
+
+.CSI restore ended with 'PartiallyFailed' status and PVCs not created
+
+CSI restore ended with `PartiallyFailed` status. PVCs are not created, pod are in `Pending` status. This issue has been resolved in {oadp-short} 1.2.5.
+
+link:https://issues.redhat.com/browse/OADP-3171[(OADP-3171)]
+
+.PodVolumeBackup fails on completed pod volumes
+
+In earlier versions of {oadp-short} 1.2, when there is a completed pod that mounted volumes in a namespace used by the Restic `podvolumebackup` or Velero backup, the backup does not complete successfully. This occurs when `defaultVolumesToFsBackup` is set to `true`. This issue has been resolved in {oadp-short} 1.2.5.
+
+link:https://issues.redhat.com/browse/OADP-1870[(OADP-1870)]
+
+.Volume Snapshots are not removed when using `velero backup delete`
+
+In earlier versions of {oadp-short} 1.2, when deleting backups using `velero backup delete`, `volumeSnapshot` and `volumeSnapshotContents` are not deleted from either {OCP} (OCP) or the storage backend. This issue has been resolved in {oadp-short} 1.2.5.
+
+link:https://issues.redhat.com/browse/OADP-2396[(OADP-2396)]
```

Comment:
This issue has been marked as "Obsolete" and looks like no fixes were there . Do we want to keep it still?